planet/clients/orders.py

```diff
@@ -502,3 +509,269 @@ async def list_orders(self,
                                                params=params)
         async for o in Orders(response, self._session.request, limit=limit):
             yield o
+
+
+class OrdersAPI:
+    """Orders API client"""
+
+    _client: OrdersClient
+
+    def __init__(self, session: Session, base_url: Optional[str] = None):
+        """
+        Parameters:
+            session: Open session connected to server.
+            base_url: The base URL to use. Defaults to production orders API
+                base url.
+        """
+
+        self._client = OrdersClient(session, base_url)
+
+    def create_order(self, request: Dict) -> Dict:
+        """Create an order.
+
+        Example:
+
+        ```python
+
+        from planet import Planet, order_request
+
+        def main():
+             pl = Planet()
+             image_ids = ["20200925_161029_69_2223"]
+             request = order_request.build_request(
+                 'test_order',
+                 [order_request.product(image_ids, 'analytic_udm2', 'psscene')]
+             )
+             order = pl.orders.create_order(request)
+        ```
+
+        Parameters:
+            request: order request definition (recommended to use the order_request module to build a request)
+
+        Returns:
+            JSON description of the created order
+
+        Raises:
+            planet.exceptions.APIError: On API error.
+        """
+        return self._client.call_sync(self._client.create_order(request))
+
+    def get_order(self, order_id: str) -> Dict:
+        """Get order details by Order ID.
+
+        Parameters:
+            order_id: The ID of the order
+
+        Returns:
+            JSON description of the order
+
+        Raises:
+            planet.exceptions.ClientError: If order_id is not a valid UUID.
+            planet.exceptions.APIError: On API error.
+        """
```

Comment:
This docstring is exactly the same as the docstring for it's async counterpart, `OrdersClient().get_order()`.  I wonder if we could simplify things by applying the docstring from `OrdersClient.get_order()` to this method so that the same docstring isn't defined in two places?  By 'apply' I mean something like 'share' the docstring between the two methods rather than 'inherit' because the implementation in this MR doesn't follow a standard Python inheritance pattern; the `OrdersClient` is passed to `OrdersAPI` via the `OrdersAPI._client` property.  Here's an example of what I am thinking:

```python
class OrdersClient:
    def get_order(self):
        """Docstring for get_order()."""
        pass

class Child:
    def get_order(self):
        self.get_order.__doc__ = self._client.get_order.__doc__
        # Add any additional docstring here
        pass
```

Reply:
I tend to think the negatives outweigh the positives here. Readers of the code would not have a docstring to refer to in the usual place and we also would not be able to use examples in shared docstrings since the methods are called differently.

Reply:
Understood!  FWIW I don't think dynamic vs statically defined docstrings should block this MR.  Check out https://github.com/planetlabs/planet-client-python/pull/1077 where I implemented a dynamic approach.  If that dynamic approach passes review, we can merge to main and ship it as an enhancement at a later date.